client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -296,6 +296,23 @@ const MetadataEditor = createClass({
 
 			{this.renderRenderOptions()}
 
+			<hr/>
+
+			<h1 className='sectionHead'>Authors</h1>
+
+			{this.renderAuthors()}
+
+			<StringArrayEditor label='invited authors' valuePatterns={[/.+/]}
+				validators={[(v)=>!this.props.metadata.authors.includes(v)]}
+				placeholder='invite author' unique={true}
+				values={this.props.metadata.invitedAuthors}
+				notes={['Invited authors are case sensitive.', 'After adding an invited author, send them the edit link. There, they can choose to accept or decline the invitation.']}
```

Comment:
> There, they can choose to accept or decline the invitation.

That's the intent, but I'm going to handle that in a separate PR. For now, the users can secure their invitation by saving.